src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -26,6 +26,48 @@ public InternalPersonnelController()
         {
         }
 
+        [EmulatedUserSupport]
+        [MapToApiVersion("1.0")]
+        [MapToApiVersion("2.0")]
+        [HttpOptions("departments/{departmentString}/resources/personnel")]
+        public async Task<ActionResult<ApiCollection<ApiInternalPersonnelPerson>>> OptionsDepartmentPersonnel([FromRoute] OrgUnitIdentifier departmentString)
```

Comment:
Moved this options endpoint to separate, as it did not work in the combination with `/projects/...` endpoints. This had a required parameter for project identifier. This has probably never worked and was not in use last 30 days.